Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -7884,12 +7883,29 @@ def is_demisto_version_ge(version, build_number=''):
     :return: True if running within a Server version greater or equal than the passed version
     :rtype: ``bool``
     """
+
+    def versionzfill(v):
+        """ Split version on . and zfill. So we can compare 6.10 to 6.5 properly.
+
+        :type version: ``str``
+        :param version: Version to check
+
+        :return: True if running within a Server version greater or equal than the passed version
+        :rtype: ``bool``
```

Comment:
looks like doc comment copied form another place 
versionzfill is return tuple

Reply:
Correct. Will fix